docs/getting-started/quick-start.ipynb

```diff
@@ -155,7 +155,7 @@
    },
    "outputs": [],
    "source": [
-    "sc.plot(dataset, backend='matplotlib')"
+    "sc.plot(dataset)"
```

Comment:
@nvaytet Not here?

Reply:
correct, that one is broken!
i think we need a docs test also on travis..